api/views.py

```diff
@@ -41,6 +41,7 @@
     WorkshopsOverTimeSerializer,
     InstructorsOverTimeSerializer,
     InstructorNumTaughtSerializer,
+    InstructorsByTime,
```

Comment:
Yeah, it should perhaps be `InstructorsByTimePeriodSerializer`.

`InstructorsOverTimeSerializer` counts how many instructors there are.